documentation/book/assembly-upgrade.adoc

```diff
@@ -5,18 +5,81 @@
 [id='assembly-upgrade-{context}']
 = {ProductName} and Kafka upgrades
 
-Each version of {ProductName} supports a range of versions of Apache Kafka.
-You can upgrade to a higher Kafka version as long as that version is supported by your version of {ProductName}.
+You can upgrade {ProductName} with no cluster downtime. Each version of {ProductName} supports one or more versions of Apache Kafka. You can upgrade to a higher Kafka version as long as that version is supported by your version of {ProductName}.
 In some cases, you can also downgrade to a lower supported Kafka version.
 
-When a newer version of {ProductName} is available, it may provide support for newer versions of Kafka. 
-Therefore, you will need to upgrade to the new version of {ProductName} before you can upgrade to a higher supported Kafka version.
-Upgrading the version of {ProductName} is done by upgrading the Cluster Operator deployment to the new version.
+{ProductName} provides support for newer versions of Kafka, but you need to upgrade {ProductName} _before_ you can upgrade to a higher supported Kafka version.
+
+*Procedure outline*
+
+In summary, upgrading {ProductName} is a two-stage process. To upgrade brokers and clients without downtime, you *must* complete the upgrade procedures in the order shown here.
+
+. First, update your Cluster Operator to the latest {ProductName} version.
+
+.. xref:assembly-upgrade-cluster-operator-{context}[]
+
+. Second, upgrade all Kafka brokers and client applications to the latest Kafka version.
+.. xref:assembly-upgrading-kafka-versions-{context}[]
 
 ifdef::StrimziUpgrades[]
-include::proc-upgrading-the-cluster-operator-0-10-0-to-0-11-0.adoc[leveloffset=+1]
+== Kafka versions
+
+The Kafka message format version and inter-broker protocol version are subject to change. As a result, the upgrade process involves making configuration changes to existing Kafka brokers and client applications (consumers and producers). The following table shows the differences between the Kafka versions:
+
+include::snip-kafka-versions.adoc[leveloffset=+1]
+
+endif::StrimziUpgrades[]
+
+ifndef::StrimziUpgrades[]
+== Kafka versions
```

Comment:
This is the same heading title as the one above. Can we change this one or the one on line 25?

Reply:
You only see one due to the If statements (which will be going). The content in the assembly will be reduced too.